brave/brave-ios
# Fix #5489: Fetch custom networks list from core

## Summary of Changes
- Fetch custom networks list from core so we don't have to manually add each new preloaded networks to our known networks list anymore
- Add Aurora asset icon, update Celo network icon from `svg` to `pdf`.

This pull request fixes #5489

## Submitter Checklist:


## Test Plan:
1. Verify Aurora no longer shows in Custom networks list
2. Add some custom networks (can be added via chainlist.org)
3. Verify custom networks appear in Custom networks list
4. Verify can still remove custom networks


## Screenshots:

![network selector - aurora added](https://user-images.githubusercontent.com/5314553/181839185-052fba93-df55-490c-a8c0-e2a31e382703.png) | ![network selector - aurora added - dark mode](https://user-images.githubusercontent.com/5314553/181839190-ef592309-c8d0-4d54-85c9-714d1ea198cd.png)
--|--

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
